### ISSUE TITLE
Is it possible to add motion blur on the cursor?

### ISSUE THREAD

**Issue by LAZAR-code** [VirtCode/hypr-dynamic-cursors#16]
Just asking. :D

**Comment by VirtCode** [comment]
This would be possible, and more importantly pretty cool. But it would require the cursor to be drawn with custom shader, which I deem out of scope for this here plugin.

But thanks for the suggestion, will put it on my ideas-to-be-implemented-if-i-ever-have-too-much-time-list, together with things like cursor trails (#6).

**Comment by littleblack111** [comment]
ngl. with the stretch mode, it already look like it have motion blur. but ya motion blur would be pretty cool, and more visually satisfying

**Comment by prufaraSS** [comment]
does this mean adding like outlines or shadows would also require implementing shaders?

**Comment by VirtCode** [comment]
Yes you're right, that would also require the use of a different/custom shader. Although I gotta admit, shadows would be pretty cool as the baked-in shadows of many cursor themes are incorrect as soon as the shape is rotated. But yeah, I think that's still out of scope for now.
